common/remote/sockfile.cpp

```diff
@@ -4365,12 +4431,71 @@ class CRemoteFileServer : public CInterface, implements IRemoteFileServer, imple
             listenep.getIpText(ips);
             acceptsock.setown(ISocket::create_ip(listenep.port,ips.str()));
         }
+        if (useSSL) {
+            querySecuritySettings(&useSSL, &configepPort, &clientCertificate.certificate, &clientCertificate.privateKey);
```

Comment:
I think the dafilesrv is really the server and perhaps this could be serverCertificate ?  And then clients may not have to set this.  But need to follow existing security code path to figure this out.